src/components/about.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import aboutStyles from "../styles/about.module.css";
+import Resume from "../statics/resume.json";
+
+export default function About() {
+
+  return (
+    <div className={aboutStyles.container}>
```

Comment:
Try and see if you can setup `styled-component` with this project by following this: https://www.gatsbyjs.org/docs/styled-components/

Thus you will have more readable code.